CatalogService/CatalogService/Models/RequestModel/AddCategoryRequest.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace CatalogService.Models.RequestModel
+{
+    public class AddCategoryRequest
```

Comment:
These are not requests but request models so lets rename these classes to have the suffix "model".